diff -u -p -Nr --exclude CVS shadow-20000902.orig/libmisc/pwdcheck.c shadow-20000902/libmisc/pwdcheck.c
--- shadow-20000902.orig/libmisc/pwdcheck.c	1999-07-09 18:11:35.000000000 +0200
+++ shadow-20000902/libmisc/pwdcheck.c	2004-11-02 18:39:47.000000000 +0100
@@ -40,7 +40,8 @@ bailout:
 	retcode = pam_acct_mgmt(pamh, 0);
 	if (retcode == PAM_NEW_AUTHTOK_REQD) {
 		retcode = pam_chauthtok(pamh, PAM_CHANGE_EXPIRED_AUTHTOK);
-	} else if (retcode)
+	}
+	if (retcode)
 		goto bailout;
 
 	if (pam_setcred(pamh, 0))
